### ISSUE TITLE
'@com_google_protobuf//:protobuf_python' is duplicated in the 'deps' attribute of rule 'validate_py'

### ISSUE THREAD

**Issue by ptarjan** [protocolbuffers/protobuf#9772]
Upgrading to 3.20.0 from 3.19.4 is now erroring.

**What version of protobuf and what language are you using?**
Version: 3.20.0
Language: Many

**What operating system (Linux, Windows, ...) and version?**
Linux

**What runtime / compiler are you using (e.g., python version or gcc version)**

**What did you do?**
```
$ bazel build ...
```

**What did you expect to see**
No error

**What did you see instead?**
```
ERROR: /private/var/tmp/_bazel_paul.tarjan/16b823dd05de063e5dc3f5967533d5ec/external/com_github_envoyproxy_protoc_gen_validate/validate/BUILD:28:17: Label '@com_google_protobuf//:protobuf_python' is duplicated in the 'deps' attribute of rule 'validate_py'
```

**Anything else we should know about your project / environment**

**Comment by ptarjan** [comment]
if I put a print lower in that file https://github.com/protocolbuffers/protobuf/blob/main/protobuf.bzl#L557 I get:
```
@@ -551,6 +552,7 @@ def py_proto_library(

     if default_runtime and not default_runtime in py_libs + deps:
         py_libs = py_libs + [default_runtime]
+    print(py_libs + deps)
     py_library(
         name = name,
         srcs = outs + py_extra_srcs,
```

```
DEBUG: /private/var/tmp/_bazel_paul.tarjan/16b823dd05de063e5dc3f5967533d5ec/external/com_google_protobuf/protobuf.bzl:552:10: [Label("@com_google_protobuf//:protobuf_python"), "@com_google_protobuf//:protobuf_python"]
```

**Comment by ptarjan** [comment]
https://github.com/protocolbuffers/protobuf/commit/faa42e900cad226eec6df2781336224d0b3ea5e7 fixed it for me and it already looks like it is cherry-picked into `3.20.0-rc1`

**Comment by acozzette** [comment]
#9696 has the fix for this and it will be included in 3.20.1, so let me go ahead and close this.
